src/fontra/views/editor/scene-model.js

```diff
@@ -706,6 +713,61 @@ export class SceneModel {
   //fontGuidelineSelectionAtPoint(point, size) {
   // }
 
+  backgroundImageSelectionAtPoint(point) {
+    // TODO: If backgroundImages are locked or hidden don't allow selection
+    // if (
+    //   !this.editorController.visualizationLayersSettings.model[
+    //     "fontra.background.image"
+    //   ]
+    // ) {
+    //   return new Set();
+    // }
+
+    const positionedGlyph = this.getSelectedPositionedGlyph();
+    if (!positionedGlyph) {
+      return new Set();
+    }
+
+    const backgroundImage = positionedGlyph.glyph.backgroundImage;
+    if (!backgroundImage) {
+      return new Set();
+    }
+
+    const image = this.fontController.getBackgroundImageCached(
+      backgroundImage.identifier,
+      () => controller.requestUpdate()
+    );
+
+    if (!image) {
+      return new Set();
+    }
+
+    const x = point.x - positionedGlyph.x;
```

Comment:
Olli, please do read the feedback I wrote on the old PR. Why do you just paste the old code instead of doing it right based on my feedback? You're again reinventing transformation math badly instead of doing the right thing and using a Transform object. Convert the DecomposedTransform to a regular Transform first.

Reply:
Note that the image should also participate in rect selection, so you will need to write `backgroundImageSelectionAtRect()` as well. The two methods will have the transformed bounding box logic in common.

(Come to think of it, did we consciously decide to let anchors and guidelines _not_ participate in rect select? We may want to revisit that decision.)

Reply:
About: _(Come to think of it, did we consciously decide to let anchors and guidelines not participate in rect select? We may want to revisit that decision.)_
-> I checked RoboFont and Glyphs and both do not include anchors or guidelines via 'rect-select'.

Reply:
In this case I honestly don't care what RF or Glyphs do. So yes, I want to revisit that decision for anchors and guidelines. But first, yes, I do want `backgroundImageSelectionAtRect()`.

Reply:
OK, done.
Most of the code for `backgroundImageSelectionAtPoint` and `backgroundImageSelectionAtRect` would be identical, why I ended up to create `backgroundImageSelectionAtPointRect` instead. I hope this is also in your interest. 
I also created this issue: https://github.com/googlefonts/fontra/issues/1785 as a possible follow up for including  anchors and guidelines into rect-select. So we don't forget about it. 
I am closing this as feedback as resolved.